packaging/kibana.spec

```diff
@@ -56,6 +56,10 @@ if [ -e /usr/local/%{name}-%{kibana_version}-linux-x64/refreshKibanaIndex.sh ];
    rm /usr/local/%{name}-%{kibana_version}-linux-x64/refreshKibanaIndex.sh
 fi
 
+if [ -e /usr/local/%{name}-%{kibana_version}-linux-x64/src/public/netmon_libs/fieldMap.js ]; then
```

Comment:
The only location for this file is under `/usr/local/www/probe/skin/js/fieldMap.js`